sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapExamples.cs

```diff
@@ -53,5 +59,14 @@ public void InterestRateSwapCreationAndUpsertionExample()
             // Delete Instrument
             _instrumentsApi.DeleteInstrument("ClientInternal", uniqueId); 
         }
+        
+        [TestCase(ModelSelection.ModelEnum.SimpleStatic)]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
+        public void InterestRateSwapValuationExample(ModelSelection.ModelEnum model)
```

Comment:
This test and the one in InterestRateSwapWIthNamedConventionsExamples.cs still look identical. What is the difference?

Reply:
Yes, just pushed now. I pushed a different commit with more models first, sorry.

The diff is two different ways of creating a swap. I changed the method call inside now.